### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by tomasesquivelgc**
## In this branch:

 - CSS is added to the project.
 - CSS is bundled into a single file.
 - Added images and other resources.

### CODE REVIEW

File: src/styles/Book.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  color: #121212;
+}
+
+.title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'RoboroSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
```
Comment (1GORDON) [inline src/styles/Book.css:41]: Great job @tomasesquivelgc  🤝

- [ ] As stated in the project requirements, kindly ensure there is no repeating of the same rule. Instead, create a single class to represent this rule and reuse this class where necessary. For example make sure the following classes are not repeated. `font-family: Montserrat, sans-serif;`, `font-family: 'RoboroSlab', sans-serif;`, etc. You may implement this across all your stylesheets. You may use this [article](https://speckyboy.com/good-bad-css-practices/) for more clarity.

File: src/styles/BookList.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  border-bottom: solid 1px #e8e8e8;
+  width: 90%;
+  padding-inline-start: 0;
+  padding-top: 1px;
+  padding-bottom: 1%;
+}
+
+.list li {
+  list-style-type: none;
+}
```
Comment (1GORDON) [inline src/styles/BookList.css:17]: Great design @tomasesquivelgc 🤝

- [ ] As stated in the project requirements, we are supposed to replicate the given design as much as possible. Kindly ensure the highlighted section replicates the given design as much as possible. You may separate these elements properly.
- Your design
your-design
- Given design
Given design

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @tomasesquivelgc ✋

Good job so far!

## Highlights!
- Basic styling well implemented ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @tomasesquivelgc , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**